staging/src/k8s.io/cli-runtime/pkg/resource/helper.go

```diff
@@ -64,6 +66,11 @@ func (m *Helper) DryRun(dryRun bool) *Helper {
 	return m
 }
 
+func (m *Helper) WithFieldManager(fieldManager string) *Helper {
```

Comment:
You should probably also document this method

Reply:
Done